api/src/paths/project/{projectId}/attachments/list.test.ts

```diff
@@ -38,141 +37,11 @@ describe('lists the project attachments', () => {
     sinon.restore();
   });
 
-  it('should throw a 400 error when no sql statement returned for getProjectAttachmentsSQL', async () => {
```

Comment:
why remove this test?

Reply:
This test is moved to project service as no sql error is not thrown in list.test.ts, now it is thrown by attachment-service.